tests/func/test_studio.py

```diff
@@ -0,0 +1,105 @@
+from requests import Response
+
+from dvc.cli import main
+from dvc.commands.studio import DEFAULT_SCOPES
+from dvc.utils.studio import STUDIO_URL
+from tests.unit.command.test_studio import MOCK_RESPONSE
+
+
+def test_auth_expired(mocker, M):
```

Comment:
@amritghimire, also, please move the most of these tests to `dvc-studio-client`.
And here let's mock the studio-client's API and test for saving config.